client/examples/tutorial.rs

```diff
@@ -185,3 +189,56 @@ fn asset_registration_test(config: &Configuration) -> Result<(), Error> {
     // Finish the test successfully
     Ok(())
 }
+
+fn asset_minting_test() -> Result<(), Error> {
+    // BEGIN FRAGMENT: mint_asset
+    use std::str::FromStr;
+    use iroha_data_model::Value;
+    use iroha_data_model::IdBox;
+    use iroha_data_model::prelude::MintBox;
+    use iroha_data_model::prelude::AssetId;
+    use iroha_data_model::prelude::AssetDefinitionId;
+    use iroha_data_model::prelude::AccountId;
```

Comment:
you can combine imports:

```suggestion
use iroha_data_model::{
    IdBox, Value, prelude::{
        AccountId, AssetId, AssetDefinitionId, MintBox, 
    }
 }
```

Also you can import prelude all at once `prelude::*`.

Reply:
Thank you.